File: 99problems.js
// GitHub: EntityPlantt/Kattis
require("readline").createInterface({
	input: process.stdin,
	output: process.stdout
}).on("line", n => {
	n = parseInt(n);
	if (n < 100) {
		console.log(99);
	}
	else if (Math.floor(n / 100) * 100 + 99 - n > n - Math.floor(n / 100) * 100 + 1) {
		console.log(Math.floor(n / 100) * 100 - 1);
	}
	else {
		console.log(Math.floor(n / 100) + "99");
	}
	return 0;
});

File: 99problems.cpp
// GitHub: EntityPlantt/Kattis
#include <iostream>
using namespace std;
int main() {
    int n;
    cin >> n;
    if (n < 100) {
        cout << 99;
    }
    else if (n / 100 * 100 + 99 - n > n - n / 100 * 100 + 1) {
        cout << n / 100 * 100 - 1;
    }
    else {
        cout << n / 100 << 99;
    }
    return 0;
}